docs/domains/Domain.md

```diff
@@ -17,6 +17,7 @@ DomainSpec is a description of a domain.
 | Name | Type | Description |
 | --- | --- | --- |
 | `adminServer` | [Admin Server](#admin-server) | Configuration for the Administration Server. |
+| `allowReplicasBelowMinDynClusterSize` | Boolean | Whether to allow the number of replicas to drop below the minimum dynamic cluster size configured in the WebLogic domain home configuration, if this is not specified at the cluster level. |
```

Comment:
Add 'Defaults to true.' (Similarly add the default in any other description.)

Reply:
done. thanks